src/event.rs

```diff
@@ -661,6 +661,16 @@ impl<'a, ES: EventSet> Sender<'a, ES> {
         unsafe { self.world.queue_global(ptr, GlobalEventIdx(event_idx)) };
     }
 
+    /// Grab the state of the sender; the [`EventSet`] indices.
+    pub fn state(&self) -> &ES::Indices {
+        self.state
+    }
```

Comment:
Actually I would prefer a more specific name for this method.

```suggestion
    /// Get a reference to the [`EventSet`] indices of this sender.
    pub fn indices(&self) -> &ES::Indices {
        self.state
    }
```